contents/blog/product-market-fit-game.md

```diff
@@ -0,0 +1,351 @@
+
+* **You are talking to the wrong users.** If so, no one will take calls, or if they do, they don't have your problem. Ask yourself – why did you have this problem, but those speaking to you don't? When you do get calls, ask who would have this problem. Change the user you are targeting.
+
+* **You aren't a good listener.** You're doing calls, users are seemingly confirming that they have your problem, but they don't use the product later. Why? Ask some open-ended questions to users in each call – perhaps you're close but not _quite_ right with your problem and target user statement. Ask your co-founder to shadow you on some calls, and give each other direct feedback afterwards on which questions worked well. This may help you find a new problem, in case...
+
+* **The problem you want to solve isn't there.** Pivot. Find a new problem to solve.
+
+### Level 3 - Validate your MVP by recruiting real users
+
+//TODO: Level 3 image here
+
+You should start Level 3 as soon as you have something for users to use. You need to _keep_ talking to users throughout this level, but now you can validate better because you have a product they can see.
+
+The goal here is to see if people actually care, as quickly as possible. No matter how excited you get after interviewing people about their problems, there's no substitute for seeing what people _do_ instead of what they _say_. 
+
+With one of the ideas Tim and I worked on before we pivoted to PostHog, we had 15 companies who "wanted to try it". We built a product, sent the invite link, and only _one_ even clicked the link and started using it. It's a good thing we hadn't spent 6+ months building the "perfect" product before this happened!
+
+The most obvious users to start with here are those that you've already met during levels 1 and 2. You should have been identifying a _problem_ that these users have during those calls. Now you have a _solution_, you should email them saying so.
+
+#### Failure modes
+
+These are all the reasons why users may not use your product:
+
+* **Users have no idea what your solution is**, so they think it could be a waste of time. Even if your product is free, people's time is not. The more senior your likely users, the stronger this effect. Can you work with someone that _reports_ to the person you were targeting to get things setup? If you struggle to get responsiveness, start questioning if something else is causing issues – even a senior stakeholder will put in work to solve a serious problem they have.
```

Comment:
```suggestion
* **Users have no idea what your solution is**, so they think it could be a waste of time. Even if your product is free, people's time is not. The more senior your users, the stronger the effect. Can you work with someone that _reports_ to the person you were targeting to get things setup? If you struggle to get responses, start questioning if something else is causing issues – even a senior stakeholder will put in work to solve a serious problem they have.
```